.rustfmt.toml

```diff
@@ -1,8 +1,12 @@
 max_width = 90 # changed
+comment_width = 90 # changed
 normalize_comments = true # changed
 imports_layout = "Vertical" # changed
 imports_granularity = "Crate" # changed
 trailing_semicolon = false # changed
 edition = "2021" # changed
 use_try_shorthand = true # changed
-use_field_init_shorthand = true # changed
\ No newline at end of file
+use_field_init_shorthand = true # changed
+reorder_impl_items = true # changed
```

Comment:
Ohh, wait, it "Reorder impl items. type and const are put first, then macros and methods.".

I thought it would do the same order as in the trait definition. Then maybe it is better to remove it=) But if someone likes it, we can keep it=)